source/features/highest-rated-comment.tsx

```diff
@@ -0,0 +1,82 @@
+import './highest-rated-comment.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+
+type Props = {
+	id: string;
+	text: string;
+	avatar: HTMLImageElement;
+}
+
+const element = ({id, text, avatar}: Props): Node => (
+	<div className="timeline-comment-wrapper">
+		{avatar}
+
+		<a href={`#${id}`} className="no-underline">
+			<div className="bg-white details-reset rounded-1 rgh-highest-rated-comment">
+				<div className="bg-gray border-bottom-0 px-2 py-0">
+					<div className="d-flex flex-items-center">
+						<span className="btn btn-sm mr-2 pr-1">
+							{icons.arrowDown()}
+						</span>
+
+						<div className="text-gray timeline-comment-header-text rgh-highest-rated-comment-text">
+							Highest-rated comment: <em>{text}</em>
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+);
+
+function getCount(reaction: HTMLElement): number {
+	return Number(reaction.textContent!.match(/\d+/)![0]);
+}
+
+function init(): void {
```

Comment:
```suggestion
function init(): false | void {
```

Reply:
<img width="412" alt="" src="https://user-images.githubusercontent.com/1402241/58749068-99736600-84b3-11e9-9f18-ff05e4753dd6.png">

You can use this button to merge my suggestions as well

Reply:
Somewhat this button doesn't work for me (even on Files page) so I'll manually add it.